tests/testthat/test-hive.R

```diff
@@ -0,0 +1,172 @@
+context("Hive translation")
+
+# Initialize simulated DB connection -------------------------------------
+
+hive_connection <- dbplyr:::simulate_hive()
```

Comment:
Can you follow the updated style in `test-translate-mssql()` (including the context and the file name)